conf.py

```diff
@@ -54,7 +54,7 @@
 
 # General information about the project.
 project = u'testcontainers'
-copyright = u'2017, Sergey Pirogov'
+copyright = u'2017-2023, Sergey Pirogov and Testcontainers Python contributors'
```

Comment:
Thoughts on "since 2017" so we don't need to update again?

Reply:
Changes are fine. Convenction with current year in the description leaves best impression on new users.